src/app/Shared/Services/Api.service.tsx

```diff
@@ -361,6 +367,24 @@ export class ApiService {
       );
   }
 
+  uploadSSLCertificate(file: File): Observable<string> {
+    const body = new window.FormData();
+    body.append('cert', file);
+    return this.sendRequest('certificates', "v2", {
```

Comment:
For code style consistency the string literal `"v2"` here (and other `"v1"`s above) should use single-quotes, not double-quotes.